### PR TITLE
[TG-122][Dima D] Plug-n-Play --BE finished

### PR DESCRIPTION
**Pull request by demakoff**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/28" target="_blank">ReviewNinja

File: src/API/socket/device-events.js [hunk 1]
```diff
@@ -74,6 +77,10 @@ export default function(io){
             debug('Socket disconnected');
         }
 
-    }
+        function isAllowedEventToSend(event) {
+            let allowedEvents = ['status', 'device-add'];
 
+            return allowedEvents.indexOf(event) !== -1;
```
Comment (antonSavk) [inline src/API/socket/device-events.js]: You can use .includes() function to check if item is in the array !fix  !fix

File: src/devices/deviceConnected.js [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
     function saveDevice (data) {
-        Sensor.find({description : data.value.description}, function (error, records) {
+
+        Sensor.find({
+            mqttId : data.device
+        }, function (error, records) {
+            
             if (records.length){
                 debug('Device already added');
             }else{
```
Comment (antonSavk) [inline src/devices/deviceConnected.js]: no spaces !fix

File: src/API/socket/device-events.js (hunk 1 above)
Comment (demakoff) [inline src/API/socket/device-events.js]: Actually, I thought about it, but unfortunately, .includes() currently isn't part of specification ES6.
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

File: src/devices/deviceConnected.js (hunk 2 above)
Comment (demakoff) [inline src/devices/deviceConnected.js]: fixed, but I think that such conventions must be handled with eslint not manually.

File: src/API/socket/device-events.js (hunk 1 above)
Comment (demakoff) [inline src/API/socket/device-events.js]: !fixed

File: src/devices/deviceConnected.js (hunk 2 above)
Comment (demakoff) [inline src/devices/deviceConnected.js]: !fixed

**Comment by kosich** [comment]
⭐

**Pull request merged**